hazelcast/src/main/java/com/hazelcast/sql/impl/QueryUtils.java

```diff
@@ -52,6 +52,9 @@ public final class QueryUtils {
 
     public static final String WORKER_TYPE_STATE_CHECKER = "query-state-checker";
 
+    // Random uuid to distinguish data links in sql.catalog among other catalog items
+    public static final String DATA_LINK_KEY_PREFIX = "8bb6ac12-bd07-11ed-afa1-0242ac120002.";
```

Comment:
I am not sure if I like the UUID here. Couldn't it be simply some reserved name?

Reply:
The clash probability for 5.2 -> 5.3 transition scenario is way lower with some UUID than with some reserved & readable name.

Reply:
`__data-link.` maybe? Collisions are either way going to be possible with a fixed name prefix until we add validation to `CREATE` statement to prevent that from happening, but at least we'll have a readable name prefix until then.